### PR TITLE
Show dnf command output

### PR DESCRIPTION
**Pull request by asilveir**
Some changes to show DNF output

### CODE REVIEW

**Comment by vinzenz** [comment]
I think it'd be worth it to look into adding a parameter into [leapp.libraries.stdlib.run](https://github.com/oamg/leapp/blob/master/leapp/libraries/stdlib/__init__.py#L121) to support this out of the box and use this then.

**Comment by asilveir** [comment]
> I think it'd be worth it to look into adding a parameter into [leapp.libraries.stdlib.run](https://github.com/oamg/leapp/blob/master/leapp/libraries/stdlib/__init__.py#L121) to support this out of the box and use this then.
> 
> The logging handler then can be then configured to handle the output.

Thank you Vinny, I think it's gotten better now.

File: repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import os
 import json
+import sys
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/actor.py]: where is the import used?

**Comment by pirat89** [comment]
@asilveirI I guess that this depends on PR https://github.com/oamg/leapp/pull/453, right?

File: repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py [hunk 2]
```diff
@@ -52,9 +53,19 @@ def permission_guard():
     # FIXME: Not implemented yet. Is it even useful?
     raise NotImplementedError
 
+def _logging_handler(fd_info, buffer):
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: Why are you implementing the function? I know, it has "_" underscore, but it's completely same function as in the framework. I would think about the rename (e.g. default_handler) in the framework so there will be something default that could be used. (what do you think guys? @vinzenz, @shaded-enmity, @fernflower)

Comment (shaded-enmity) [reply repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: +1 on providing something like a `standard_io_handler` for these purposes

Comment (jmikovic) [reply repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py]: can this be done in follow-up PR?
I would like this to be merged ASAP as it is quite difficult to debug errors in transaction

**Comment by Rezney** [comment]
As per @jmikovic comment, I would also prefer this to be done sooner then later.

**Comment by asilveir** [comment]
> missing sys import in https://github.com/oamg/leapp-repository/blob/10cdaac215fb5ff4694fc9cc43a4e7176ddcfb58/repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py

Done

**Pull request merged**